src/setup/rgw.py

```diff
@@ -14,77 +14,221 @@
 # For any questions about this software or licensing,
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 
+import os
+import time
+import uuid
+import errno
+import shutil
+import socket
+from cortx.utils.validator.v_pkg import PkgV
+from cortx.utils.conf_store import Conf, MappedConf
+from cortx.utils.conf_store.error import ConfError
+from cortx.utils.process import SimpleProcess
+from cortx.utils.log import Log
+from src.setup.error import RgwSetupError
+from src.setup.const import (
+    cortx_rpms, ceph_rpms, ceph_conf_tmpl_file, ceph_conf_tmpl_url,
+    ceph_conf_file, ceph_conf_path, ceph_admin_keyring_file)
+
 
 class Rgw:
     """Represents RGW and Performs setup related actions."""
 
+    _machine_id = Conf.machine_id
+
     @staticmethod
     def validate(phase: str):
-        """Perform validtions."""
+        """Perform validations."""
+
+        Log.info(f'validations started for {phase} phase.')
+
+        if phase == "post_install":
```

Comment:
make it single quote to maintain consistency. 
"post_install" --> 'post_install'